doctr/models/recognition/crnn/tensorflow.py

```diff
@@ -34,6 +34,14 @@
                   'kçHëÀÂ2É/ûIJ\'j(LNÙFut[)èZs+&°Sd=Ï!<â_Ç>rêi`l'),
         'url': 'https://github.com/mindee/doctr/releases/download/v0.1.1/crnn_resnet31-69ab71db.zip',
     },
+    'crnn_mobilenet_small': {
```

Comment:
same here could you rename it? (and the TF version of `mobilenetv3_small` --> `mobilenet_v3_small` so that it's rather uniform between frameworks :pray: )